pkg/resources/services/service.go

```diff
@@ -157,7 +151,9 @@ func NewHeadlessForLog(t *miniov1.Tenant) *corev1.Service {
 // NewClusterIPForConsole will return a new cluster IP service for Console Deployment
 func NewClusterIPForConsole(t *miniov1.Tenant) *corev1.Service {
 	consolePort := corev1.ServicePort{Port: miniov1.ConsolePort, Name: miniov1.ConsoleServicePortName}
-	consoleTLSPort := corev1.ServicePort{Port: miniov1.ConsoleTLSPort, Name: miniov1.ConsoleServiceTLSPortName}
+	if t.TLS() {
+		consolePort = corev1.ServicePort{Port: miniov1.ConsoleTLSPort, Name: miniov1.ConsoleServiceTLSPortName}
+	}
```

Comment:
@Alevsk want to confirm if this change is fine for console?

Reply:
@nitisht Here 
```
if t.TLS() || t.ConsoleExternalCert() {
		consolePort = corev1.ServicePort{Port: miniov1.ConsoleTLSPort, Name: miniov1.ConsoleServiceTLSPortName}
}
```

Would be better, since TLS() internally could be just `t.ExternalCert()` and if that's the case console will not be deployed with TLS enabled